package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hrs-cordova-plugin-twilio",
-  "version": "3.0.3",
+  "version": "3.0.5",
```

Comment:
Changed to 3.0.5 as Oscar's MR for twilio upgrade has changed it to 3.0.4 already.